frontend/src/apis/index.js

```diff
@@ -9,9 +9,13 @@ const myFetch = async (url, config = {}) => {
         credentials: 'same-origin',
         body: JSON.stringify(config.body),
     });
-    if (response.status >= 200 && response.status < 300)
-        return await response.json();
-    else throw new Error('Error in making request');
+    if (response.status >= 200 && response.status < 300) {
+        try {
+            return await response.json();
+        } catch (error) {
```

Comment:
Shouldn't the error be thrown here instead of just failing silently?

Reply:
I will change all the responses to json.

Right now what we have is in the below format
```res.status(200).send(user);```

After change
```res.status(200).json(user);```

Would this be OK?.

Reply:
Sounds good :)